packages/jest-types/src/Matchers.ts

```diff
@@ -5,7 +5,7 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-// TODO: Move this to `expect` when it's migrated
+// TODO: Remove this when whole project is migrated
```

Comment:
cannot do it currently, because of `export =` stuff, and importing from `expect/src/types` seemed fishy (and caused TS errors, oh well who's got time for that XD)